src/app/features/hiro-messages/in-app-messages.tsx

```diff
@@ -7,6 +7,7 @@ import { useDismissedMessageIds } from '@app/store/settings/settings.selectors';
 
 import { HiroMessageItem } from './components/in-app-message-item';
 
+//  TODO test this as per PR instructions
```

Comment:
What does this comment mean here?

Reply:
Kyran gave me some notes on how to test `InAppMessages` still work by altering the config but I lost them in the comments 😢  

Do you know how to do this? Basically I want to make sure that if I need to add an `InAppMessages` that the UI looks OK

Reply:
See the `wallet-config.md` file for instructions